### DISCUSSION TITLE
Git Error: fatal: protocol error: bad line length 196 send-pack: unexpected disconnect while reading sideband packet

### DISCUSSION THREAD

**Discussion by Emmykage** [community/community#113521] [Programming Help]
### Body

Please i keep getting this error on my terminal. I have no idea what it means even though I have tried different solutions and none has worked.

```
fatal: protocol error: bad line length 196
send-pack: unexpected disconnect while reading sideband packet
error: failed to push some refs to 'https://github.com/xxxxxx/xxxxxx.git'
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Obad94** [comment]
**Increase Buffer Size:**
You can try increasing the Git buffer size by running the following command:
`git config --global http.postBuffer 524288000`
